Update results paragraph (#39)
greenelab/xswap-manuscript: content/02.body.md

We find superior performance in predicting the co-authorship relationships (AUROC 0.75), which was expected as the network being predicted shared roughly the same degree distribution as the network on which the edge prior was computed.
The results of the third prediction task show that a difference in degree distribution between the network on which features are computed and the network to be predicted can make prediction significantly more challenging.

> The edge prior can be considered a baseline edge predictor that excellently captures degree's contribution to the probability of an edge existing.

Reviewer: ```suggestion
The edge prior can be considered a baseline edge predictor that accurately captures degree's contribution to the probability of an edge existing.
```

The edge prior's low performance in the third task indicates that degree is less helpful for edge prediction tasks in which training and testing networks do not share their degree distributions.
Many biomedical prediction tasks can be framed as edge prediction tasks between different degree distributions.

> In drug repurposing, for example, existing compound-disease treatment relationships are not randomly sampled from all true treatment relationships.

Reviewer: ```suggestion
In drug repurposing, for example, existing compound-disease treatment relationships are unlikely to be randomly sampled from all true treatment relationships.
```

However, all treatment relationships between existing compounds and diseases are desirable outputs in prediction.

> Framed as edge prediction in this way, drug repurposing attempts to predict between different degree distributions.

Reviewer: ```suggestion
Understanding the extent to which degree influences accuracy metrics can reveal the extent to which predictions from one or more networks are due to underlying biological properties, as opposed to network degree distributions, which may be consistent and generalize to the true network which has a different distribution.
```
I do not really like this - but I think it's the sentiment that's missing.

Author: I agree that this sentiment is a good way to conclude this section and motivate the section that follows. How about the following:

```suggestion
Edge predictions can be based on both underlying biological properties and network degree distributions.
However, predictions based on biological properties may be more consistent and generalizable than those based on degree.
Degree's influence on edge prediction accuracy measures can reveal the relative contributions of these two factors.
```

Reviewer: I agree with the change above 👍
